### DISCUSSION TITLE
Cannot find it in the extension list

### DISCUSSION THREAD

**Discussion by vini-brito** [community/community#13359] [Copilot Conversations] [answered]
I paste in the quick action "ext install GitHub.copilot" and it tries to find it in the marketplace but nothing comes up.
I try searching for related keywords and I see lots of other "github" related extensions but not this one.

Here is what I see: 

![image](https://user-images.githubusercontent.com/6878457/159764791-248e36bb-b794-40b3-8577-45269c95938d.png)

**Comment by davecheney** [comment, marked as answer]
I'm sorry you are having difficulties. Copilot currently supports VSCode. We don't support VSCodium or CodeOSS.
